src/shearwater_predator_parser.c

```diff
@@ -704,6 +714,14 @@ shearwater_predator_parser_cache (shearwater_predator_parser_t *parser)
 				}
 			}
 		} else if (type >= LOG_RECORD_CLOSING_0 && type <= LOG_RECORD_CLOSING_7) {
+			if (type == LOG_RECORD_CLOSING_6) {
+				if (logversion >= 11) {
+					if (stack_time_total_s > 0) {
+						stack_time_remaining_s = array_uint16_be(data + offset + 3);
+						dc_field_add_string_fmt(&parser->cache, "Remaining stack time at end [hh:mm]", "%d:%02d:%02d", stack_time_remaining_s / 3600, (stack_time_remaining_s / 60) % 60, stack_time_remaining_s % 60);
```

Comment:
Shouldn't this be hh:mm:ss as the others?

Reply:
Correct, thanks, fixed.